src/main/java/org/killbill/billing/plugin/notification/setup/EmailNotificationListener.java

```diff
@@ -282,17 +282,20 @@ private void sendEmailForPayment(final Account account, final ExtBusEvent killbi
                     emailContent = templateRenderer.generateEmailForSuccessfulPayment(account, invoice, context);
                 } else {
                     subscriptionClient.sendEmailRequest("Purchase_Success", invoice, account);
+                    logService.log(LogService.LOG_INFO, String.format("Generated successful purchase receipt for invoice: %s", invoiceId.toString()));
                 }
             } else if (lastTransaction.getTransactionType() == TransactionType.PURCHASE && lastTransaction.getTransactionStatus() == TransactionStatus.PAYMENT_FAILURE) {
                 if (oneTimePayment) {
                     emailContent = templateRenderer.generateEmailForFailedPayment(account, invoice, context);
                 } else {
                     subscriptionClient.sendEmailRequest("Purchase_Failure", invoice, account);
+                    logService.log(LogService.LOG_INFO, String.format("Generated failed purchase receipt for invoice: %s", invoiceId.toString()));
```

Comment:
do we want to just add a `return` after the success/failure logs, to be safe? doesn't matter to me

Reply:
decided I don't care. merging